### DISCUSSION TITLE
[General] [http client] communication error: 400 Bad Request, JSON does not validate. Violations:, "Telefone" does not match to ^(Unmanaged|Computer|Phone|NetworkEquipment|Printer)$ at #->properties:itemtype

### DISCUSSION THREAD

**Discussion by tulhaum** [glpi-project/glpi-agent#805] [General]
### Your question

Hi folks!
Using GLPI-Agent 1.11 with proxy server plugin works fine inventoring computer machines.
But when I try to inventory a smartphone, GLPI Agent response: 
"[http client] communication error: 400 Bad Request, JSON does not validate. Violations:, "Telefone" does not match to ^(Unmanaged|Computer|Phone|NetworkEquipment|Printer)$ at #->properties:itemtype"
I´m using android-inventory-agent version 1.6.0, Android 14. Maybe the issue is related to the phone language in PT-BR. "Telefone" is the translation for "Phone" in pt-br.
Anyone can help  ?

**Comment by g-bougard** [comment]
@stonebuzz, have you an answer for @tulhaum ?

**Comment by stonebuzz** [comment]
Indeed, maybe a bad translation of `Phone`  for `itemtype` option!

@tulhaum Can you post here Android Agent configuration ?

**Reply by tulhaum** [reply]
![image](https://github.com/user-attachments/assets/3461a569-7ec8-499c-a3b9-cb922c4bb90f)

**Reply by stonebuzz** [reply]
A fix is currently in development.

https://github.com/glpi-project/android-inventory-agent/pull/409

**Reply by tulhaum** [reply]
I found another translation issue within 'Show my inventory' at first screen:

![image](https://github.com/user-attachments/assets/43b7d6e0-1a62-4f96-932d-d3c836cb3332)

**Reply by stonebuzz** [reply]
thanks for feedback

Another fix is currently in development.

https://github.com/glpi-project/android-inventory-agent/pull/410
